src/types/express.d.ts

```diff
@@ -1,4 +1,4 @@
-import winston from 'winston';
+import type winston from 'winston';
```

Comment:
Also a good practice to explicitly state that this is a type you are importing (enforced by an ESLint rule as well)